app/routes.py

```diff
@@ -0,0 +1,302 @@
+from flask import Blueprint, jsonify, make_response, request
+from flask.globals import _request_ctx_stack
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timedelta
+import requests
+
+customer_bp = Blueprint('customers', __name__, url_prefix='/customers')
+video_bp = Blueprint('videos', __name__, url_prefix='/videos')
+rental_bp = Blueprint('rentals', __name__, url_prefix='/rentals')
+
+# VIDEO ENDPOINTS
+
+@video_bp.route('', methods=['GET', 'POST'])
+def handle_videos():
+    if request.method == 'GET':
+        videos = Video.query.all()
+        videos_list = [{"id": video.id,
+                        "title": video.title,
+                        "release_date": video.release_date,
+                        "total_inventory": video.total_inventory
+                        } for video in videos]
+        return jsonify(videos_list)
+    elif request.method == 'POST':
+        request_body = request.get_json()
+        if 'title' not in request_body.keys():
+            return make_response({"details": "Request body must include title." }, 400)
+        elif 'release_date' not in request_body.keys():
+            return make_response({"details":"Request body must include release_date."}, 400)
+        elif 'total_inventory' not in request_body.keys():
+            return make_response({"details": "Request body must include total_inventory."}, 400)
```

Comment:
you could create a function to handle these guard clauses dynamically and then use that function for customers and videos it could look something like
```
def validate_parameters(request_body, parameters):
           for attribute in parameters:
                if attribute not in request_body:
                       abort(make_response({"details": f"Request body must include {attribute}."}, 400))

```